user_manual/files/using_e2ee.rst

```diff
@@ -0,0 +1,277 @@
+=============================================
+Using end-to-end encryption
+=============================================
+
+If enabled server side, Nextcloud provides the possibility to encrypt your files on your devices.
+This is called end-to-end encryption, or E2EE, because the files are encrypted on your device and only decrypted on your device.
+This means that the server never sees the unencrypted files.
+
+Enabling E2EE
+-------------
+
+If your administrator enabled the end-to-end encryption app, you can start using it from one of your devices.
+Open the settings of the Nextcloud client and look for the EE2E encryption settings.
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-setup.png
+    :width: 750px
+    :alt: Setup E2EE on the desktop client
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Android</summary>
+
+.. image:: ../images/e2ee-android-setup.png
+    :width: 400px
+    :alt: Setup E2EE on Android
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>iOS</summary>
+
+.. image:: ../images/e2ee-ios-setup.png
+    :width: 400px
+    :alt: Setup E2EE on iOS
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Web</summary>
+
+.. warning::
+    It is not possible to enable E2EE in the browser.
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+    <br />
+
+Encrypting a folder
+-----------------------
+
+.. warning::
+
+    You can only encrypt empty folders.
+
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-encrypt.png
+    :width: 750px
+    :alt: Encrypting a folder on the desktop client
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Android</summary>
+
+.. image:: ../images/e2ee-android-encrypt.png
+    :width: 400px
+    :alt: Encrypting a folder on an Android device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>iOS</summary>
+
+.. image:: ../images/e2ee-ios-encrypt.png
+    :width: 400px
+    :alt: Encrypting a folder on an iOS device
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+   <details>
+   <summary>Web</summary>
+
+.. warning::
+
+    It is not possible to encrypt a folder in the browser.
+
+.. raw:: html
+
+   </details>
+
+
+.. raw:: html
+
+    <br />
+
+Adding an E2EE device
+---------------------
+
+.. raw:: html
+
+   <details>
+   <summary>Desktop</summary>
+
+.. image:: ../images/e2ee-desktop-add.png
```

Comment:
- [ ] Black box used as censoring should simply be made the same colour as the UI.